examples/destination/java/src/main/java/destination/DestinationServiceImpl.java

```diff
@@ -15,32 +18,50 @@ public void configurationForm(ConfigurationFormRequest request, StreamObserver<C
+                                        .build(),
                                 FormField.newBuilder()
-                                        .setName("region").setLabel("AWS Region").setRequired(false).setDropdownField(
-                                                DropdownField.newBuilder().addAllDropdownField(
-                                                        Arrays.asList("US-EAST", "US-WEST")).build()
-                                        ).build(),
+                                        .setSingle(Field.newBuilder().setName("region").setLabel("AWS Region").setDefaultValue("US-EAST").setRequired(false)
+                                                .setDropdownField(DropdownField.newBuilder().addAllDropdownField(Arrays.asList("US-EAST", "US-WEST")).build())
+                                                .build())
+                                        .build(),
                                 FormField.newBuilder()
-                                        .setName("hidden").setLabel("my-hidden-value").setTextField(TextField.Hidden)
+                                        .setSingle(Field.newBuilder().setName("hidden").setLabel("my-hidden-value").setTextField(TextField.Hidden).build())
                                         .build(),
                                 FormField.newBuilder()
-                                        .setName("isPublic")
-                                        .setLabel("Public?")
-                                        .setDescription("Is this public?")
-                                        .setToggleField(ToggleField.newBuilder().build())
+                                        .setSingle(Field.newBuilder().setName("isPublic").setLabel("Public?").setDescription("Is this public?")
+                                                .setToggleField(ToggleField.newBuilder().build())
+                                                .build())
                                         .build()
-                        ))
-                        .addAllTests(Arrays.asList(
+                        )).addAllTests(Arrays.asList(
                                 ConfigurationTest.newBuilder().setName("connect").setLabel("Tests connection").build(),
                                 ConfigurationTest.newBuilder().setName("select").setLabel("Tests selection").build()))
                         .build());
 
         responseObserver.onCompleted();
     }
 
+    @Override
+    public void capabilities(CapabilitiesRequest request, StreamObserver<CapabilitiesResponse> responseObserver) {
+        responseObserver.onNext(
+                CapabilitiesResponse.newBuilder()
+                        .setSupportsHistoryMode(false)
+                        .addAllDataTypeMappings(Lists.newArrayList(
+                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.STRING).setMapTo(DestinationType.newBuilder().setName("VARCHAR").setMapTo(DataType.STRING).build()).build(),
```

Comment:
We ask the partners to map all data types, so we should be doing the same in the example

Reply:
added mappings for all data types